dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBSchemaInitializer.java

```diff
@@ -378,12 +399,31 @@ private XmlSchemaElement createXsElement(XmlSchema schema,
     }
 
     public void end(FaultInfo fault) {
+
+	if (isLoggableFine) { // Liberty change begin
+	   LOG.fine("end: Processing Fault: " + fault.getFaultName());
```

Comment:
Do we need to null check `faultinfo`?

Reply:
No, the next statement after this trace references fault, so it would NPE if fault is null even without this trace.